blacklight-cornell/app/models/book_bag.rb

```diff
@@ -39,11 +39,13 @@ def create_table
       client.query("CREATE TABLE IF NOT EXISTS \
         book_bags(bagname varchar(255), bibid int unsigned, created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP, PRIMARY KEY (bagname, bibid))")
     rescue Mysql2::Error => e
-      save_level = Rails.logger.level; Rails.logger.level = Logger::WARN
-      Rails.logger.warn "jgr25_log #{__FILE__} #{__LINE__} #{__method__}: in BookBag"
-      puts e.error.to_yaml
-      puts e.errno.inspect
-      Rails.logger.level = save_level
+      # :nocov:
+        save_level = Rails.logger.level; Rails.logger.level = Logger::WARN
```

Comment:
I'm not sure that this save_level paradigm is needed. A `warn` log message should get recorded unless we have the prod log level set to `error` (in which case, it should maybe be dropped to `warn`?). If we do need it, maybe it should be refactored into LoggingHelper, as it appears in several places.